docs/gettingstarted/structuring.md

```diff
@@ -1,54 +1,83 @@
+
+    package com.examplemod.awesomesauce;
+        
+    import net.minecraft.init.Blocks;
+    import net.minecraftforge.fml.common.Mod;
+    import net.minecraftforge.fml.common.Mod.EventHandler;
+    import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+        
+    @Mod(modid = "awesomesauce", name = "AwesomeSauce", version = "0.1")
+    public class AwesomeSauceMod
+    {
+        @Instance
+        public static AwesomeSauceMod instance = new AwesomeSauceMod();
+        
+        @EventHandler
+        public void init(FMLInitializationEvent event)
+        {
+            // some example code
+            System.out.println("DIRT BLOCK >> "+Blocks.DIRT.getUnlocalizedName());
+        }
+    }
 
 What is `@Mod`?
 -------------
 
-This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
+`@Mod` is the annotation indicating that the class is a *Mod entry point* for Forge Mod Loader. It contains various bits of Metadata that the mod will use as well as modifiers that tell Forge how to handle your mod.
 
-You can find an example mod in the [Forge src download](http://files.minecraftforge.net/).
+Now, the `@Mod` annotation can take quite a few arguments, but the only three *required* ones are `modid`, `name`, and `version`.
+* `modid` is the unique identifier and *internal* name of the mod, and takes a `String`. Usually, this is the same name as the mod, but in all lowercase. It's used internally by Forge and by other Mod Developers that wish to find your mod through Forge (for instance, to use your APIs). 
+ * **ONCE YOU PICK THIS IDENTIFIER, YOU SHOULD NOT CHANGE IT DURING THE LIFE OF THE MOD. BAD THINGS HAPPEN IF YOU DO AND OTHER MODS DEPEND ON YOUR MOD.**
+* `name` is the *external* name of the mod, and takes a `String`.
+* `version` is the version of the mod, and takes a `String`. There is no set standard for how you version things, but it's recommended that you read the [versioning](https://mcforge.readthedocs.io/en/latest/conventions/versioning/) article.
 
-Keeping Your Code Clean Using Sub-packages
-------------------------------------------
+Simple, right? `@Mod` has a wide range of other things you can throw at it, but those will be covered in the articles that use them.
 
-Rather than clutter up a single class and package with everything, it is recommended you break your mod into subpackages.
+What is `@Instance`?
+-------------------
+`@Instance` is actually a child annotation of `@Mod`. It basically just gives Forge and other Mod Developers a way to instance your mod if they need to.
```

Comment:
From the JavaDocs:

    /**
      * Populate the annotated field with the mod instance based on the specified ModId. This can be used
      * to retrieve instances of other mods.
      * @author cpw
    **/

That's what I based it on, however I'm also not entirely sure what it does anyways, so I'll look around and learn about it and rewrite this section after I educate myself.